src/app/login-page/login-page.component.scss

```diff
@@ -0,0 +1,82 @@
+.login {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    flex: 1;
+    min-height: 100%;
+    height: 100%;
+}
+
+.login-container {
+    align-self: center;
+    width: 368px;
+    height: 286px;
+    border-radius: 6px;
+    box-shadow: 0 1px 24px 0 rgba(18, 21, 35, 0.08);
+    background-color: #ffffff;
+}
+
+h3 {
+    margin-left: 24px;
+    margin-top: 24px;
+}
+
+.login-form {
+    display: flex;
+    flex-direction: column;
+    padding: 0 24px;
+}
+
+:host {
+    height: 100%;
+    display: flex;
+    flex: 1;
+    flex-direction: column;
+    justify-content: center;
+}
+
+.btn {
+    width: 68px;
+    height: 36px;
+    background: #9bc837;
+    border-radius: 3px;
+    font-family: "Source Sans Pro", sans-serif;
+    font-weight: 600;
+    color: #ffffff;

```

Comment:
Please, try css variables for colors
https://developer.mozilla.org/ru/docs/Web/CSS/Using_CSS_custom_properties